.github/workflows/test.yaml

```diff
@@ -24,28 +24,29 @@ jobs:
     env:
       RP_SIDEKICK_DATABASE: postgres://postgres:postgres@localhost/rp_sidekick
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Install gettext
         run: sudo apt-get install gettext
-      - uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ hashFiles('requirements.txt', 'requirements-dev.txt') }}-pip
-      - uses: actions/setup-python@v2
+      - uses: actions/setup-python@v4
         with:
           python-version: 3.9
-      - name: Install dependancies
+      - uses: abatilo/actions-poetry@v2.0.0
+        with:
+          poetry-version: "1.8.3"    
```

Comment:
There is a version 1.8.4, any reason for going with the previous one? I see its only 9 days old which might be the reason 🙂

Reply:
The important thing is that the version we use here is compatible with whatever version generated the lockfile, and I think we set a specific version because there was a problem at some point. Perhaps we should look at removing the explicit version and using the latest instead?

Reply:
I prefer pinning just to avoid any issues later on.